en/flying/fixed_wing_takeoff.md

```diff
@@ -0,0 +1,56 @@
+# Fixed Wing Takeoff
+
+PX4 supports fixed-wing takeoff in [missions](#mission) and using the [Takeoff](#takeoff) flight mode.
+Vehicles can be *catapult/hand-launched* or use the runway takeoff mode. <!-- runway support in missions? -->
+In all cases the vehicle takes off at a predefined pitch in its current direction (RC stick input is ignored).
+
+The sections below explain the main methods.
+
+
+## Takeoff Flight Mode {#takeoff}
+
+[Takeoff Mode](../flight_modes/takeoff.md#fixed_wing) enables takeoff using either *catapult/hand-launch* (default) or *runway takeoff*.
+
+### Catapult/Hand-Launch Mode
+
+When armed and in takeoff mode the vehicle waits to detect launch (i.e. from acceleration due to catapult or hand launch). 
+On launch detection, the vehicle ramps up to to maximum throttle ([RWTO_MAX_THR](#RWTO_MAX_THR)) in about 2 seconds and then performs a full throttle climbout at a 10 degree pitch.
+The vehicle climbout phase ends once the vehicle reaches the correct altitude (defined by [FW_CLMBOUT_DIFF](#FW_CLMBOUT_DIFF)), after which regular navigation proceeds.
```

Comment:
1. FW_CLMBOUT_DIFF - I understand what this means for a mission - it is how close you have to get to the takeoff item altitude before you assume that you have reached it. But what does it mean for manual mode, where there is no target altititude? Or is the altitude at which climbout ends in this case RWTO_NAV_ALT ?
2. For Fixed wing, we just go into Loiter after takeoff in manual mode right?

Reply:
> For Fixed wing, we just go into Loiter after takeoff in manual mode right?

I'm confused.  manual mode is "manual".  there is no "go into Loiter after takeoff" when manual flying.  If you take your hands of the sticks the vehicle just does whatever aerodynamics dictates....

Reply:
@Antiheavy This was meant to be your question "https://github.com/PX4/px4_user_guide/pull/744/files#r433008339". Essentially "in takeoff mode" what happens when you get to to the end of climbout. I presume loiter, but need that confirmed, along with confirmation of what height that occurs at.

Sorry about using "manual" terminology

Reply:
@hamishwillee In manual modes there is the concept of "takeoff help". If you takeoff in a manual mode then the target altitude is set to `takeoff_alt + fw_climbout_diff`.

Reply:
@RomanBapst So you're kind of saying that in a mission `fw_climbout_diff` is used as an acceptence distance from your target altitude, while if no target altitude is set (for example in takeoff mode) then `fw_climbout_diff` becomes the target altitude?